### PR TITLE
Fix storage would return wrong int vid ending with '\0'

### LINKED ISSUE #170: VertextID normalize for int type.
Currently, VID allows both fixed_string and integer types.
we need to normalize the return value for MetaClient.

for example :

```
int64_t vid = 1, should be return "\001\000\000\000\000\000\000"
fixed_string vid = "a\000\000\000\000\000\000" length is 8 , should be return "a"
```

### PR DESCRIPTION
**Pull request by critical27**
Rely on https://github.com/vesoft-inc/nebula-common/pull/263.

When the vid is integer, it could end with '\0', we can't simply discard them. So we need to be aware of vid type. It involves as follows:
1. GetNeighbors/GetProp/Update
2. StorageExpressionContext
3. Index, (not done, wait https://github.com/vesoft-inc/nebula-storage/pull/171 merge at first)

Fix unstable ut in ci as well.

close #170, #166, #161

### CODE REVIEW

File: src/storage/context/StorageExpressionContext.cpp [hunk 1]
```diff
@@ -65,9 +66,11 @@ Value StorageExpressionContext::getEdgeProp(const std::string& edgeName,
             return Value::kNullValue;
         }
         if (prop == kSrc) {
-            return NebulaKeyUtils::getSrcId(vIdLen_, key_);
+            auto srcId = NebulaKeyUtils::getSrcId(vIdLen_, key_);
+            return isIntId_ ? srcId : srcId.subpiece(0, srcId.find_first_of('\0'));
```
Comment (laura-ding) [inline src/storage/context/StorageExpressionContext.cpp:70]: user can insert vid such as "aa\0bb". so graphd need to check vid if has `\0` and return an error to the user.

Comment (critical27) [reply src/storage/context/StorageExpressionContext.cpp:70]: Yeah, graph need to prevent '\0' for string id.

**Review by bright-starry-sky — APPROVED** [review]
LGTM. excellent

**Review by panda-sheep — DISMISSED** [review]
Well done! Excellent!

File: src/storage/BaseProcessor.h [hunk 2]
```diff
@@ -56,6 +56,13 @@ class BaseProcessor {
             return cpp2::ErrorCode::E_SPACE_NOT_FOUND;
         }
         spaceVidLen_ = len.value();
+
+        auto vIdType = this->env_->schemaMan_->getSpaceVidType(spaceId);
+        if (!vIdType.ok()) {
+            return cpp2::ErrorCode::E_SPACE_NOT_FOUND;
+        }
+        isIntId_ = vIdType.value() == meta::cpp2::PropertyType::INT64 ? true : false;
```
Comment (liuyu85cn) [inline src/storage/BaseProcessor.h:64]: isIntId_ = vIdType.value() == meta::cpp2::PropertyType::INT64

Comment (critical27) [reply src/storage/BaseProcessor.h:64]: My bad, good catch!

**Review by bright-starry-sky — APPROVED** [review]
LGTM.

**Review by panda-sheep — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]

**Review by liuyu85cn — APPROVED** [review]

**Pull request merged**